Src/AutoFixtureUnitTest/Kernel/TerminatingWithPathSpecimenBuilderTest.cs

```diff
@@ -162,35 +159,62 @@ public void SpecimenRequestsWillInitiallyBeEmpty()
         [Fact]
         public void SpecimenRequestsRaisedFromTracerAreRecordedCorrectly()
         {
-            // Fixture setup
-            var tracer = new DelegatingTracingBuilder();
-            var specimens = new[] { new object(), new object(), new object() };
-            var requestEvents = specimens.Select((o, i) => new RequestTraceEventArgs(o, i)).ToList();
-            var sut = new TerminatingWithPathSpecimenBuilder(tracer);
-            // Exercise system
-            requestEvents.ForEach(tracer.RaiseSpecimenRequested);
-            // Verify outcome
-            Assert.True(specimens.SequenceEqual(sut.SpecimenRequests));
-            // Teardown
+            // arrange
```

Comment:
BTW, in such cases you can leave a single comment, so I'll find all the places and fix that. It will be simpler for you and for me to close each one later 😅